From ecd4e67a28dd5ff7cdc4f91dc0562704035dffcc Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Tue, 30 Jun 2026 19:29:44 -0700
Subject: [PATCH 13100/15333] apparmor: reserve mediation class for packet
 mediation

Packet mediation is going to be added in the future, reserve a class
for it.

Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 security/apparmor/audit.c            | 2 +-
 security/apparmor/include/apparmor.h | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/security/apparmor/audit.c b/security/apparmor/audit.c
index f22dca9b0cc3..c32828f55bcc 100644
--- a/security/apparmor/audit.c
+++ b/security/apparmor/audit.c
@@ -67,7 +67,7 @@ static const char *const aa_class_names[] = {
 	"unknown",
 	"unknown",
 	"unknown",
-	"unknown",
+	"netv9_packet",
 	"X",
 	"dbus",
 };
diff --git a/security/apparmor/include/apparmor.h b/security/apparmor/include/apparmor.h
index cc6e3df1bc62..cb06b2f55d49 100644
--- a/security/apparmor/include/apparmor.h
+++ b/security/apparmor/include/apparmor.h
@@ -36,6 +36,7 @@
 #define AA_CLASS_NS		21
 #define AA_CLASS_IO_URING	22
 
+#define AA_CLASS_NETV9_SKB	30
 #define AA_CLASS_X		31
 #define AA_CLASS_DBUS		32
 
-- 
2.55.0

